### PR TITLE
Support SPANNER_EMULATOR_HOST for admin client

### PR DESCRIPTION
**Pull request by yfuruyama**
[DatabaseAdminClient](https://godoc.org/cloud.google.com/go/spanner/admin/database/apiv1#DatabaseAdminClient) doesn't support `SPANNER_EMULATOR_HOST` environment variable to decide which Spanner endpoint is used, so I added the code to check `SPANNER_EMULATOR_HOST` internally to switch the endpoint. This is a similar way to google-cloud-go integration [test](https://github.com/googleapis/google-cloud-go/blob/master/spanner/integration_test.go#L162).

Tested with [handy-spanner](https://github.com/gcpug/handy-spanner).

### CODE REVIEW

File: integration_test.go [hunk 1]
```diff
@@ -91,6 +91,10 @@ func setup(t *testing.T, ctx context.Context, dmls []string) (*Session, string,
 		t.Fatalf("failed to create table: err=%s", err)
 	}
 
+	// Wait until created table becomes visible.
+	// TODO(furuyama): Change to a sophisticated way for checking table creation
+	time.Sleep(10 * time.Second)
```
Comment (hengfengli) [inline integration_test.go]: Just curious: is there a chance that when `op.Wait(ctx)` is finished, the created table is still not visible?

Comment (yfuruyama) [reply integration_test.go]: I didn't dig into the details, but it looked so.

Without time.Sleep(10 sec), I got a following error occasionally at touching the database.
`spanner: code = "Aborted", desc = "Transaction aborted. Database schema probably changed during transaction, retry may succeed."`

Comment (hengfengli) [reply integration_test.go]: Ok. I see. If that is the case, adding a retrying here probably would be a bit nice. But you can do it later.

Comment (yfuruyama) [reply integration_test.go]: Sure, I'll change that to do retry in the next pull request. Thank you!

**Pull request merged**